src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -0,0 +1,2565 @@
+import logging
+import json
+import tempfile
+import random
+import threading
+import subprocess
+import socket
+import time
+import os
+from http import server as http_server
+from random import randint
+from .tests import get_realm, \
+    ZonegroupConns, \
+    zonegroup_meta_checkpoint, \
+    zone_meta_checkpoint, \
+    zone_bucket_checkpoint, \
+    zone_data_checkpoint, \
+    zonegroup_bucket_checkpoint, \
+    check_bucket_eq, \
+    gen_bucket_name, \
+    get_user, \
+    get_tenant
+from .zone_ps import PSTopic, \
```

Comment:
dont import from zone_ps (you should import the XXXS3 classes from the new file)

Reply:
Yeah I'll change that